asm/multiboot_header.asm

```diff
@@ -14,9 +19,19 @@ header_start:
     ; checksum
     dd 0x100000000 - (MULTIBOOT2_MAGIC_NUMBER + 0 + (header_end - header_start))
 
+    ; framebuffer setup
+framebuffer_tag_start:
+    dw MULTIBOOT_HEADER_TAG_FRAMEBUFFER
+    dw MULTIBOOT_HEADER_TAG_OPTIONAL
+    dd framebuffer_tag_end - framebuffer_tag_start
```

Comment:
Do we really need this? I thought we could avoid `framebuffer_tag_*`.

Reply:
Unfortunately we need it for now since it's pretty complicated to set up a framebuffer manually. GRUB takes care of this for us.